ui/App/ProjectScreen/Source/Commit.svelte

```diff
@@ -72,6 +79,38 @@
     border-top: 1px solid var(--color-foreground-level-3);
     margin: 1rem 0 1.5rem 0;
   }
+
+  .anchor {
+    background-color: var(--color-foreground-level-1);
+    border-radius: 0.5rem;
+    border: 1px solid var(--color-foreground-level-2);
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  .header {
+    border-bottom: 1px solid var(--color-foreground-level-2);
+    min-height: 3.5rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 1rem 0 1rem;
+  }
+
+  .timestamp {
+    color: var(--color-foreground-level-5);
+  }
+
+  .metadata {
```

Comment:
This class is already applied to another element in the slot for the `BackButton` component. Is this intentional? I feel that `anchor-transaction-hash` would be more descriptive.

Reply:
Wasn't intentional. Seems like the previous `metadata` class wasn't used.

aa9bc1f7422719b8afef06ffe2faeb48cffa0f75